### ISSUE TITLE
Force Enable Backdrop in Jellyfin 10.9.6

### ISSUE THREAD

**Issue by sannidhyaroy** [BobHasNoSoul/jellyfin-mods#27]
In Jellyfin's earlier versions (don't exactly remember the version number), I tried the mod from this repository to force enable backdrop in Jellyfin and it worked. However, after updating to Jellyfin 10.9.6, I see this wasn't working, so I thought that the update might have rewritten the mod back to the initial code. But I found that my changes were still there, so I started to edit the return value and found that it works with either `P` or `x`.

So, in summary, for Jellyfin 10.9.6, to enable backdrop by default for new users, you ensure this string `enableBackdrops:function()` is set to return either `P` or `x` like:
```js
enableBackdrops:function(){return P}
```

**Comment by BobHasNoSoul** [comment]
I will look into this, when I'm free from work

**Comment by BobHasNoSoul** [comment]
sorry for the late response, between work life and being a dad this side project got put on the back burner.. but here is the answer to your question :) 

## Force Backdrops for all users (10.9.x)

in main.jellyfin.bundle.js simply search for `enableBackdrops:function(){return _}` and replace it with `enableBackdrops:function(){return E}` save and then you need reload your cache and reload the page.. if trying to do this without a full cache chear go to your.url/web/main.jellyfin.bundle.js and then press crtl+shift+r (in most browsers this clears it)


let me know how it goes so i can close this issue :)

**Comment by sannidhyaroy** [comment]
This works as well. I noted you can replace `_` with either `E`, `P`, or `x`, and all three of them works.
